lib/deploy.js

```diff
@@ -33,6 +34,20 @@ module.exports = promisify(function(config, callback) {
   var rootDir = config.rootDir ? config.rootDir : '.';
   var cloneDir = config.cloneDir ? config.cloneDir : null;
 
+  if (config.fileGlobs) {
+    fs.mkdirSync('tmp');
```

Comment:
Can we use the *temp* directory here to create and (automatically) destroy a temp directory that is guaranteed not to already exist?